.github/workflows/build_and_test.yml

```diff
@@ -77,8 +77,51 @@ jobs:
       run: echo "::error file=.github/workflows/build_and_test.yml::File not included in any filter" && false
       if: ${{ !contains(steps.filter.outputs.*, 'true') }}
 
-  rust:
-    name: Rust
+  rust-stable:
+    name: Rust stable
+
+    runs-on: ubuntu-latest
+
+    needs: changes
+
+    if: ${{ needs.changes.outputs.rust == 'true' }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: sudo apt-get install gcc-multilib
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        components: rustfmt, clippy
+        target: i686-unknown-linux-gnu
+
+    - name: Check for duplicate dependencies
+      run: |
+        DUPS="$(cargo tree -d -e normal --workspace)"
+        echo "$DUPS"
+        test -z "$DUPS"
+
+    - name: Build
+      run: cargo +stable build --verbose
+
+    - name: Run tests
+      run: cargo +stable test --workspace --verbose -- --include-ignored
+
+    - name: Run tests (32-bit)
+      # Exclude device-transfer because OpenSSL, used for reference results,
+      # doesn't support the --target option.
+      # Exclude signal-neon-futures because those tests run Node
+      run: cargo +stable test --workspace --verbose --target i686-unknown-linux-gnu --exclude device-transfer --exclude signal-neon-futures -- --include-ignored
```

Comment:
I believe signal-neon-futures isn't in the default set anyway.

Reply:
I copied that from the nightly script. Still want me to drop it?

Reply:
Yes please, it falls out of `--all` vs. `--workspace`.